.github/workflows/changelog-check.yml

```diff
@@ -5,9 +5,6 @@ on:
   pull_request:
     paths:
       - 'version.txt'
-  push:
-    paths:
-      - 'version.txt'
```

Comment:
why do we remove it?

Reply:
We have no need to run this on push anymore. Only PR 🙌
I discussed this with @michpohl recently.